flytepropeller/pkg/controller/nodes/executor.go

```diff
@@ -790,6 +795,34 @@ func (c *nodeExecutor) preExecute(ctx context.Context, dag executors.DAGStructur
 	return handler.PhaseInfoNotReady("predecessor node not yet complete"), nil
 }
 
+// checkOffloadingCompat checks if the upstream and downstream nodes are compatible with the literal offloading feature
+func (c *nodeExecutor) checkOffloadingCompat(ctx context.Context, nCtx interfaces.NodeExecutionContext, inputLiterals map[string]*core.Literal, node v1alpha1.ExecutableNode) handler.PhaseInfo {
+	isOffloadLiteral := false
```

Comment:
```suggestion
	consumesOffloadLiteral := false
```

or something else you'd prefer, only because we can have multiple literals here and this var name sounds like there is just one

Reply:
done